WD-Firmware/examples/build/SConscript_freertos

```diff
@@ -39,26 +34,34 @@ list_files_c=[
    [os.path.join(rtos_base, 'tasks.c'), os.path.join(out_dir, 'tasks.o')],
    [os.path.join(rtos_base, 'timers.c'), os.path.join(out_dir, 'timers.o')],
    #[os.path.join(rtos_base, 'portable', 'MemMang', 'heap_4.c'), os.path.join(out_dir, 'heap_4.o')],
-] + list_port_files_c[Env['TARGET_BOARD']]
+] + list_port_files_c
+
+list_files_a=[
+   [os.path.join(rtos_base, 'portable', 'portASM.S'), os.path.join(out_dir, 'portASM.o')],
+]
+
+
+list_port_include_paths=[
+  os.path.join(Env['ROOT_DIR'], 'rtos', 'rtosal', 'config', Env['TARGET_BOARD'] ),     

```

Comment:
1. rtosasl will push this to public 
2. all SS should remove this "list_port_include.."